app/main.py

```diff
@@ -1,8 +1,26 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        self.people.update({
+            name: self
+        })
```

Comment:
Updating the `people` dictionary with `self.people.update({name: self})` might lead to overwriting existing entries if a `Person` with the same name is created again. Consider checking if the name already exists in the dictionary before updating.